app/src/main/res/layout/fragment_about_us.xml

```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragment.AboutUsFragment">
+
+  <ScrollView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content">
+      <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:orientation="vertical">
+          <LinearLayout
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:layout_marginTop="5dp"
+              android:layout_marginLeft="5dp"
+              android:layout_marginRight="5dp"
+              android:background="@color/colorPrimaryDark"
+              android:orientation="horizontal">
+              <ImageView
+                  android:layout_width="50dp"
+                  android:layout_height="50dp"
+                  android:layout_gravity="center"
+                  app:srcCompat="@drawable/ic_version_white_24dp"
+                  />
+              <LinearLayout
+                  android:layout_width="match_parent"
+                  android:layout_height="wrap_content"
+                  android:layout_margin="5dp"
+                  android:orientation="vertical">
+                  <TextView
+                      android:layout_width="wrap_content"
+                      android:textSize="20sp"
+                      android:textColor="@color/mb_white"
+                      android:layout_marginStart="20dp"
+                      android:gravity="center"
+                      android:layout_height="match_parent"
+                      android:text="@string/version_text"/>
+                  <TextView
+                      android:layout_width="wrap_content"
+                      android:textSize="15sp"
+                      android:textColor="@color/mb_white"
+                      android:layout_marginStart="20dp"
+                      android:gravity="center"
+                      android:layout_height="match_parent"
+                      android:text="@string/version_value"/>
```

Comment:
Don't add this as a contant. See this https://stackoverflow.com/a/4929559/3623128